gr-blocks/python/blocks/msg_pair_to_var.py

```diff
@@ -24,13 +27,17 @@ def __init__(self, callback):
         self.set_msg_handler(pmt.intern("inpair"), self.msg_handler)
 
     def msg_handler(self, msg):
-        try:
-            new_val = pmt.to_python(pmt.cdr(msg))
+        if not pmt.is_pair(msg) or pmt.is_dict(msg) or pmt.is_pdu(msg):
+            gr.log.warn("Input message %s is not a simple pair, dropping" % repr(msg))
+            return
 
+        print("calling " + repr(self.callback.__name__) + " with " + repr(new_val))
```

Comment:
log.debug()?

Reply:
Thanks. Debug statement that was not removed....fixed.